src/domains/wallet-common/env/weu-dev/terraform.tfvars

```diff
@@ -0,0 +1,66 @@
+prefix         = "pagopa"
+env_short      = "d"
+env            = "dev"
+domain         = "wallet"
+location       = "westeurope"
+location_short = "weu"
+instance       = "dev"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "Dev"
+  Owner       = "pagoPA"
+  Source      = "https://github.com/pagopa/pagopa-infra/tree/main/src/domains/wallet-common"
+  CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+}
+
+### External resources
+
+monitor_resource_group_name                 = "pagopa-d-monitor-rg"
+log_analytics_workspace_name                = "pagopa-d-law"
+log_analytics_workspace_resource_group_name = "pagopa-d-monitor-rg"
+
+### Aks
+
+ingress_load_balancer_ip = "10.1.100.250"
+
+external_domain          = "pagopa.it"
+dns_zone_internal_prefix = "internal.dev.platform"
+
+### Cosmos
+
+cosmos_mongo_db_params = {
+  enabled      = true
+  kind         = "MongoDB"
+  capabilities = ["EnableMongo", "EnableServerless"]
+  offer_type   = "Standard"
+  consistency_policy = {
+    consistency_level       = "BoundedStaleness"
+    max_interval_in_seconds = 5
+    max_staleness_prefix    = 100000
+  }
+  server_version                   = "4.0"
+  main_geo_location_zone_redundant = false
+  enable_free_tier                 = true
+
+  additional_geo_locations          = []
+  private_endpoint_enabled          = false
+  public_network_access_enabled     = true
+  is_virtual_network_filter_enabled = false
+
+  backup_continuous_enabled = false
+
+}
+
+cidr_subnet_cosmosdb_wallet = ["10.1.153.0/24"]
+cidr_subnet_redis_wallet    = ["10.1.148.0/24"]
+cidr_subnet_storage_wallet  = ["10.1.154.0/24"]
```

Comment:
Maybe those have to differs from the one from ecommerce domain?

Reply:
No should be changed with those specific to the new domain

Reply:
subnet updated